SPECS/patterns-ceph-containers/patterns-ceph-containers.spec

```diff
@@ -0,0 +1,102 @@
+Summary:        Patterns for the Ceph containers
+Name:           patterns-ceph-containers
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Metapackages
+URL:            http://en.opensuse.org/Patterns
+Source0:        %{name}-rpmlintrc
+ExclusiveArch:  x86_64 aarch64 ppc64le s390x
+
+%description
+This is an internal package that is used to create the patterns as part
+of the installation source setup. Installation of this package does
+not make sense.
+
+%package ceph_base
+Summary:        Ceph base
+Group:          Metapackages
+Provides:       pattern() = ceph_base
+#Provides:      pattern-icon() = yast-iscsi-client
```

Comment:
I'm assuming this is just temporary for the draft phase but to be sure: let's remove commented sections in the final version.